.github/workflows/script_checks.yml

```diff
@@ -0,0 +1,21 @@
+name: 🤹 Script checks
+
+on:
+  push:
+    branches:
+      - master
+      - release-**
+  pull_request:
+  release:
+    types: ['prereleased', 'released']
```

Comment:
I know it's not changing with this PR but shouldn't we have `[published]` here instead to avoid that a prerelease published from a draf does not trigger the workflow?
See https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

Reply:
Yes, done in 5105ff4da74fd7b56430e7856f96bcac99565597

Reply:
You can remove `released` as well (glaubs)..